tests/Functional/Schema/OracleSchemaManagerTest.php

```diff
@@ -276,4 +276,17 @@ public function testQuotedTableNameRemainsQuotedInSchema(): void
         $columns = $schemaManager->listTableColumns('"tester"');
         self::assertCount(1, $columns);
     }
+
+    public function testReservedTableNameCommentRetrieval(): void
```

Comment:
Please try to make it a non-Oracle specific test. Even though you found the issue on Oracle, we want the expected behavior to be enforced on all platforms.

There are test you can try updating instead: https://github.com/doctrine/dbal/blob/851e6be197dd0f4cd6a0c22182caa1f8f6aa0178/tests/Functional/Schema/SchemaManagerFunctionalTestCase.php#L1204-L1225

Reply:
I updated the test, also I added a reserved column name so it is more complete.